api/mailer/templates/verification.js

```diff
@@ -0,0 +1,23 @@
+const generateHashedId = require('../auth').generateHashedId
+
+module.exports = function (user, recipient, name) {
+  return new Promise((resolve, reject) => {
+    generateHashedId(recipient)
+      .then(hashedId => {
+        return resolve({
+          from: user,
+          to: recipient,
+          subject: 'Please verify your email address',
+          generateTextFromHTML: true,
+          html: `
+            Hi ${name},<br />
+            <p>Please verify your email by clicking below</p>
+            <a href='https://sce.engr.sjsu.edu/verify?id=${hashedId}&user=${recipient}'>Verify Email</a>
```

Comment:
Let's also create a page on the website that the link directs to, It could be something very simple, like the [ConfirmationPage component](https://github.com/SCE-Development/Core-v4/blob/dev/src/Pages/MembershipApplication/ConfrimationPage.js).

Reply:
Created a verifyEmail component and added it to the /verify route